pkg/storage/local/impl/store.go

```diff
@@ -111,10 +111,6 @@ const (
 		drop table if exists rigs;
 		drop table if exists rig_parts;
 		drop table if exists rig_deals;
-		// TODO: Investigate what we want here.
-		// drop table if exists cids;
-		// drop table if exists table_names;
-		// drop table if exists txns;
 	`
```

Comment:
Can we just move the comments outside of the string literal so we don't loose that TODO?
```suggestion
	`
	// TODO: Investigate what we want here.
	// drop table if exists cids;
	// drop table if exists table_names;
	// drop table if exists txns;
```